tests/wpt/web-platform-tests/html/semantics/forms/form-submission-0/submit-entity-body.html

```diff
@@ -5,14 +5,46 @@
 <script>
 var simple_tests = [
   {
-    name: "form submission should navigate to url with x-www-form-urlencoded",
+    name: "form submission from form should navigate to url with x-www-form-urlencoded",
     input: "<input name=foo value=bara>",
-    enctype: "application/x-www-form-urlencoded"
+    enctype: "application/x-www-form-urlencoded",
+    submitelement: "",
+    submitaction: function(doc) { doc.getElementById("testform").submit(); }
```

Comment:
Could this pass in the element that should be used for submitting instead (i.e. instead of passing in a doc, pass in testform)?